src/components/gitOps/UserGitRepConfiguration.tsx

```diff
@@ -108,7 +108,7 @@ const UserGitRepConfiguration: FunctionComponent<UserGitRepoConfigurationProps>
     return (
         <div className="w-100 h-100 bcn-0 pt-16 flexbox-col">
             <div className="w-960">
-                <div className="fs-16 fcn-9 fw-6 ml-20 mb-8">GitOps Configuration</div>
+                <div className="fs-16 fcn-9 fw-6 ml-20 mb-8" data-testid="gitops-config-heading">GitOps Configuration</div>
```

Comment:
Why do we need test id on a heading? It doesn't seem to be a interactive element